src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -153,61 +154,84 @@ const FormattedReport = ({ summaries, weekIndex }) => {
     }
   };
 
+  const bioSwitch = (userId, bioPosted) => {
+    const [isBioPosted, setIsBioPosted] = useState(bioPosted);
+    return (
+      <div>
+        <div className="bio-toggle">
+          <b>Bio announcement:</b>
+        </div>
+        <div className="bio-toggle">
+          <ToggleSwitch
+            switchType="bio"
+            state={isBioPosted ? false : true}
+            handleUserProfile={() => {
+              handleChangeBioPosted(userId, isBioPosted);
+              setIsBioPosted(!isBioPosted);
+            }}
+          />
+        </div>
+      </div>
+    );
+  };
+
+  const bioLabel = (userId, bioPosted) => {
+    return (
+      <div>
+        <b>Bio announcement:</b>
+        {bioPosted ? ' Posted' : ' Requested'}
+      </div>
+    );
+  };
+
+  const summaryCard = (summary, index, bioFunction) => {
+    const hoursLogged = (summary.totalSeconds[weekIndex] || 0) / 3600;
+    const googleDocLink = getGoogleDocLink(summary);
+    return (
```

Comment:
Maybe the `bioFunction` can be defined here between the lines 189 and 190 rather than having it be be a prop.